tests/test_dataclasses.py

```diff
@@ -2269,20 +2269,43 @@ class TestPymatgenFromStructureData:
     """
 
     @skip_pymatgen
-    def test_1(self):
+    @pytest.mark.parametrize(
+        'pbc', ((True, True, True), (True, True, False), (True, False, False), (False, False, False))
+    )
+    def test_1(self, pbc):
```

Comment:
While you are here, it would be nice to use more descriptive names for the test functions.